### ISSUE TITLE
Support emojis with the ZWJ

### ISSUE THREAD

**Issue by afmenez** [SeungheonOh/EmojiTools#5]
The current implementation supoorts only single codepoint emojis.  It would be nice to support emojis with the ZWJ connector, like all the flags,  the gender-specific "professions", the skin tones, and so on.
Most of the [Emoji 4.0](https://emojipedia.org/emoji-4.0/) needs the ZWJ.

**Comment by SeungheonOh** [comment]
it it possible, but the algorithm would be slower that it was may I could make a new brach for that

**Comment by afmenez** [comment]
I believe supporting all the chars is way more important than the performance in this case.

**Comment by SeungheonOh** [comment]
Okay, I'll try it

**Comment by SeungheonOh** [comment]
Yes, just made a branch that uses external text file to load emojis, there was significant performance decrease,,
